docs/manual/build_system/prerequisites.qbk

```diff
@@ -101,8 +101,8 @@ favorite compiler with __hpx__ visit __stellar_hpx_buildbot__.
 
 [note You need to build the following Boost libraries for __hpx__:
       Boost.DateTime, Boost.Filesystem, Boost.ProgramOptions, Boost.Regex,
-      Boost.Serialization, Boost.System, Boost.Thread, Boost.Chrono (starting
-      Boost 1.49.0), and Boost.Atomic (starting Boost 1.53.0). ]
+      Boost.Serialization, Boost.System, Boost.Thread, Boost.Chrono, and
```

Comment:
We don't depend on Boost.Serialization anymore. I know this is unrelated to this PR, however would you mind removing this as well?